client/jetpack-cloud/sections/overview/primary/overview-products/jetpack-products.ts

```diff
@@ -0,0 +1,49 @@
+import { translate } from 'i18n-calypso';
+
+export interface ProductData {
+	description: string;
+	url: string;
+	urlText?: string;
+	product_slug?: string;
+	data?: any;
+	name?: string;
```

Comment:
We don't appear to use these?

Reply:
`data`, `name` and `slug` are used. `urlText` is the one I have to remove.